web/src/test/java/org/openmrs/web/filter/update/GZIPFilterTest.java

```diff
@@ -9,9 +9,13 @@
  */
 package org.openmrs.web.filter.update;
 
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 import static org.mockito.Mockito.mock;
 
+import javax.servlet.FilterChain;
```

Comment:
@achilep  looks good though it came with some small code formats cc @teleivo

Reply:
that is according to our style guide